src/libstd/sys/sgx/abi/entry.S

```diff
@@ -121,6 +121,15 @@ sgx_entry:
     fnstcw %gs:tcsls_user_fcw
 /*  reset user state */
     cld /* x86-64 ABI requires DF to be unset at function entry/exit */
+
+/*  making sure AC flag is not set in rflags */
+/*  avoid using the 'clac' instruction to be compatible with older compilers */
```

Comment:
older *CPUs*

Reply:
clac was introduced in 5th generation Intel CPUs, SGX in 6th generation, so this isn't an issue